src/containers/Application.js

```diff
@@ -38,15 +38,20 @@ class Application extends Component {
 
 
 		// Instantiate modules
+		console.log ("IIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIII")
```

Comment:
Be more specific about the log message, which component it is located in etc. Obviously, user will not want to see it unless there is some error.